core/core.go

```diff
@@ -238,6 +243,23 @@ func (n *IpfsNode) startOnlineServicesWithHost(ctx context.Context, host p2phost
 	return nil
 }
 
+func (n *IpfsNode) getCacheLifetime() (time.Duration, error) {
+	cfg, err := n.Repo.Config()
+	if err != nil {
+		return 0, err
+	}
+
+	ct := cfg.Ipns.ResolveCacheTime
```

Comment:
we can, but we dont need to. It assumes that an empty value means the default value of one minute.